src/main/java/Deadline.java

```diff
@@ -0,0 +1,22 @@
+public class Deadline extends Task{
+    protected String time;
+    protected String desc;
+
+    public Deadline(String description){
+        super(description);
+        String arr[] = description.split("/", 2);
+        String arr1[] = arr[0].split(" ", 2);
```

Comment:
The standard states the array specifiers must be attached to the type not the variable.
Might be good to change this to "String[] arr" instead.
https://se-education.org/guides/conventions/java/basic.html#types